charts/generic-service/templates/scheduled-downtime-cronjob.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.scheduledDowntime.enabled -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "generic-service.fullname" . | trunc 43 }}-shutdown
+  labels:
+    {{- include "generic-service.labels" . | nindent 4 }}
+spec:
+  schedule: {{ .Values.scheduledDowntime.shutdown }}
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 345600 # 4 days
+      template:
+        spec:
+          containers:
+            - name: kubectl
+              image: bitnami/kubectl:latest
```

Comment:
We should change this to use our own image: https://github.com/ministryofjustice/hmpps-tools-images/pkgs/container/hmpps-devops-tools